library/spdm_responder_lib/libspdm_rsp_digests.c

```diff
@@ -143,8 +143,11 @@ libspdm_return_t libspdm_get_response_digests(void *context, size_t request_size
                                                response_size, response);
     }
 
-    libspdm_set_connection_state(spdm_context,
-                                 LIBSPDM_CONNECTION_STATE_AFTER_DIGESTS);
+    if (spdm_context->connection_info.connection_state <
+        LIBSPDM_CONNECTION_STATE_AUTHENTICATED) {
```

Comment:
With this, LIBSPDM_CONNECTION_STATE_AFTER_CERTIFICATE will become LIBSPDM_CONNECTION_STATE_AFTER_DIGESTS.

I dont feel good. We should maintain the LIBSPDM_CONNECTION_STATE_AFTER_CERTIFICATE status as well.